src/mimir_coordinator.py

```diff
@@ -49,8 +49,9 @@
 deployment to be considered robust according to the official recommendations/guidelines."""
 
 
-def _endpoint_to_role(endpoint: str) -> MimirRole:
-    return MimirRole(endpoint.replace("-", "_"))
+def _relation_to_role(relation: Relation) -> MimirRole:
+    # TODO: extract the role from the relation
+    return MimirRole(relation.role) # FIXME
```

Comment:
Should we get this "for free" from the pydantic deserialization?

Reply:
Actually yes; this PR should wait for the interface to be formalized so we can import it and use it here